### ISSUE TITLE
Exception in thread "main" java.lang.reflect.InvocationTargetException

### ISSUE THREAD

**Issue by Manu936** [kaz-Anova/StackNet#70]
![screenshot 42](https://user-images.githubusercontent.com/39614748/42236965-0e58201e-7f1a-11e8-8254-c9517cd208c1.png)
![screenshot 41](https://user-images.githubusercontent.com/39614748/42236971-13a2b55c-7f1a-11e8-8c70-231c48979251.png)

I know that this issue's been already addressed I tried to resolve it in by mentioning -Xmx8048m but still I am getting the same error.
I have got an 8gb RAM and Java 1.8 64bit version.

**Comment by Manu936** [comment]
XgboostClassifier booster:gbtree objective:reg:linear num_round:1000 eta:0.05 threads:1 gamma:5.0 max_depth:10 subsample:0.8 colsample_bytree:0.75 seed:1 verbose:true
XgboostClassifier booster:gblinear objective:reg:gamma num_round:500 eta:0.5 threads:1 lambda:1 alpha:1 seed:1 verbose:true

RandomForestClassifier estimators=1000 rounding:10 threads:2 max_depth:10 max_features:0.75 min_leaf:20.0 Objective:ENTROPY gamma:0.001 row_subsample:1.0 verbose:false copy=false

"this my params.txt"

**Comment by Manu936** [comment]
Oops! found it I haven't concatenated the target labels of the training set to the X_train.csv, it woks fine now.
